lib/Capabilities/CapabilitiesProcedural.js

```diff
@@ -9,8 +9,9 @@ import { columnTranslations } from '../constants/translations';
 
 import css from './styles.css';
 import ItemActionCheckbox from './ItemActionCheckbox';
+import { isUniqueField } from '../utils/isUnique';
 
-const CapabilitiesProcedural = ({ content, readOnly, onChangeCapabilityCheckbox, isCapabilitySelected, isCapabilityDisabled }) => {
+const CapabilitiesProcedural = ({ content, readOnly, onChangeCapabilityCheckbox, isCapabilitySelected, isCapabilityDisabled, capabilitiesToCompare }) => {
```

Comment:
The same as above. Each params should be set into the new line for better readability

Reply:
Fixed! Thanks for noticing